versions/draft-3/SPEC.md

```diff
@@ -1439,12 +1439,12 @@ workflow foo {
 
 ```
 $wf_parameter_meta = 'parameter_meta' $ws* '{' ($ws* $wf_parameter_meta_kv $ws*)* '}'
-$wf_parameter_meta_kv = $identifier $ws* '=' $ws* $string
+$wf_parameter_meta_kv = $identifier $ws* '=' $ws* $meta_value
 ```
 
-This purely optional section contains key/value pairs where the keys are names of parameters and the values are string descriptions for those parameters.
+This purely optional section contains key/value pairs where the keys are names of parameters and the values are JSON like expressions that describe those parameters.
 
-> *Additional requirement*: Any key in this section MUST correspond to a worflow input
+> *Additional requirement*: Any key in this section MUST correspond to an input or output parameter.
```

Comment:
If we're opening this section up to output info (which I like) then `parameter` feels like the wrong word to me now